proposals/0186-loaded-transaction-data-size-specification.md

```diff
@@ -105,14 +105,12 @@ mapping is enabled, this SIMD may be amended to count them differently.
 We include programdata size for LoaderV3 programs because using the program
 account on a transaction forces an unconditional load of programdata to compile
 the program for execution. We always count it, even when the program account is
-not a transaction program ID, because the program must be available for CPI.
+not a transaction program ID, because the program must be available for CPI. If
+the programdata account does not exist, then its size is 0, and transaction
+loading continues as normal.
 
 There is no special handling for any account owned by the native loader,
-LoaderV1, or LoaderV2.
-
-Account size for programs owned by LoaderV4 is left undefined. This SIMD should
-be amended to define the required semantics before LoaderV4 is enabled on any
-network.
+LoaderV1, LoaderV2, or LoaderV4.
```

Comment:
@Lichtso if you could confirm my understanding is correct

Reply:
Yes, only loader-v3 had an indirection. All other loaders (including v4) store the executable directly in the program account.